onnxruntime/core/providers/rknpu/shaper.h

```diff
@@ -0,0 +1,135 @@
+#pragma once
+
+#include <string>
+#include <vector>
+#include <map>
+
+#include <iostream>
+
+namespace onnxruntime {
+namespace rknpu {
+
+class Shaper {
```

Comment:
Documentation.
Lots of functions lack constness which may/will lead to const_casts and hard to find bugs.

Reply:
do you mean the parameter of functions?